README.md

```diff
@@ -0,0 +1,87 @@
+# irma_sms_issuer
+The IRMA SMS issuer takes care of issuing a mobile phone number to [Yivi app](https://github.com/privacybydesign/irmamobile) users. It consists of a Java backend API, which connects to an [irma server (issuer)](https://github.com/privacybydesign/irmago) and an SMS gateway service, and a frontend web app.
+
+# Running (development)
+The easiest way to run the irma_sms_issuer for development purposes is by having an Android phone with the Yivi app installed and Docker.
+
+### Set up local SMS messaging gateway (Android only):
+Make sure your development machine and phone are on the same network. Install [StartHere SMS Gateway App](https://m.apkpure.com/starthere-sms-gateway-app/com.bogdan.sms) on your Android device. When the app is started, it runs a local server imitating an SMS sending gateway. SMS messages, in the form of POST requests coming from irma_sms_issuer, are sent to this messaging service and will be displayed inside the app. 
```

Comment:
I didn't like downloading some APK for this, so I tested this by catching the traffic locally in the terminal using:
```
while true; do { echo -e 'HTTP/1.1 200 OK\r\n'; sh test;  } | nc -l 8766; done
```

Reply:
Nice one! Another simplification